docs/plugins/inputs/s3.asciidoc

```diff
@@ -56,6 +58,7 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 | <<plugins-{type}s-{plugin}-session_token>> |<<string,string>>|No
 | <<plugins-{type}s-{plugin}-sincedb_path>> |<<string,string>>|No
 | <<plugins-{type}s-{plugin}-temporary_directory>> |<<string,string>>|No
+| <<plugins-{type}s-{plugin}-watch_for_new_files>> | <boolean,boolean>|No
```

Comment:
Incorrect link format for `boolean,boolean`

Reply:
Solved, in this PR and fixed in the plugin repo: https://github.com/logstash-plugins/logstash-input-s3/commit/dff182d0703cd5f8fd19ff30e6cf61b7bdefb435